test/check-machines-create

```diff
@@ -148,6 +148,14 @@ class TestMachinesCreate(VirtualMachinesCase):
                                                                          create_and_run=False),
                                              {"source-url": "Installation source must not be empty"})
 
+        # When switching from PXE mode to anything else make sure that the source input is empty
+        runner.checkDialogFormValidationTest(TestMachinesCreate.VmDialog(self, storage_size=1,
```

Comment:
This seems useful, but should be an independent commit.

Reply:
That test is identical to the one above that. I guess I accidentally doubled it when rewriting some stuff and it got into my commit.